src/libraries/System.Text.Json/tests/System.Text.Json.Tests/JsonNode/JsonArrayTests.cs

```diff
@@ -196,6 +196,48 @@ public static void CopyTo()
             Assert.Throws<ArgumentOutOfRangeException>(() => jArray.CopyTo(arr, -1));
         }
 
+        [Fact]
+        public static void ConvertJSONArrayToIListOfJsonNode()
```

Comment:
Nit: 
```suggestion
        public static void ConvertJsonArrayToIListOfJsonNode()
```

Reply:
The uppercase was to distinguish actual JSON from the "JsonArray" class